tokio/src/io/driver/mod.rs

```diff
@@ -260,7 +261,7 @@ cfg_rt! {
         /// flag is not enabled.
         pub(super) fn current() -> Self {
             crate::runtime::context::io_handle()
-                .expect("there is no reactor running, must be called from the context of Tokio runtime")
+                .unwrap_or_else(|| panic!(context_missing_error(&[])))
         }
```

Comment:
I think this is the spot where we should panic with a different error in the case where a Runtime does exist, but `enable_io` was not called.